### PR TITLE
Add input filter mode

### PR DESCRIPTION
**Pull request by limakzi**
Signed-off-by: Kamil Zabielski <kamil.zabielski@sysdogs.com>

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -6,4 +6,4 @@ export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
 find "${INPUT_PATH:-'.'}" -not -path "${INPUT_EXCLUDE}" -type f -name "${INPUT_PATTERN:-'*'}" -print0 \
     | xargs -0 misspell -locale="${INPUT_LOCALE}" -i "${INPUT_IGNORE}" \
-    | reviewdog -efm="%f:%l:%c: %m" -name="misspell" -reporter="${INPUT_REPORTER:-github-pr-check}" -level="${INPUT_LEVEL}"
```
Comment (haya14busa) [inline entrypoint.sh:9]: Please bring back `-efm` flag

Comment (limakzi) [reply entrypoint.sh:9]: @haya14busa I am not sure, why I removed it. 👯 
Done.

**Review by haya14busa — APPROVED** [review]
Thanks! :+1:

**Comment by review-dog** [comment]
Hi, @limakzi! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintainance of reviewdog.

Thanks again!

**Pull request merged**